sdk/src/main/java/com/uid2/sdk/UID2.kt

```diff
@@ -0,0 +1,30 @@
+package com.uid2.sdk
+
+/**
+ * The structure representing a version of the SDK.
+ */
+data class Version(val major: Int, val minor: Int, val patch: Int)
+
+object UID2 {
+    private const val VERSION_STRING = "0.1.0"
```

Comment:
At the moment, this is hardcoded (in this one place). As part of the changes required to support deployment to maven, we may be able to move this into Gradle so that the output `aar` file will be named correctly. We will still have to manually bump it during a release process, but having a single place will avoid any confusions.